container/example-config/basic.toml

```diff
@@ -0,0 +1,27 @@
+[[agent]]
+id = "test agent"
+name = "Holo Tester"
+key_file = "holo_tester.key"
+
+[[dna]]
+id = "app spec rust"
+file = "app-spec-rust.hcpkg"
+hash = "Qm328wyq38924y"
+
+[[instance]]
+id = "app spec instance"
+dna = "app spec rust"
+agent = "test agent"
+[instance.logger]
+type = "simple"
+file = "app_spec.log"
+[instance.storage]
+type = "file"
+path = "app_spec_storage"
+
+
+[[interface]]
+type = "websocket"
+port = 8888
+[[interface.instance]]
+id = "app spec instance"
```

Comment:
Why is interface not included with the instance config? Seems like a one-to-one relationship. Then we don't even need to refer to an id.

Reply:
I don't think it is a one-to-one relationship. I think it will be a rather regular case to have:
* an admin interface that can install/manage apps and also access every app
* an interface without admin rights that access all instances of the local user
* a few special interfaces (for command line scripts for instance) that have access to a distinct subset of instances